packages/eslint-config-react-app/index.js

```diff
@@ -155,6 +155,15 @@ module.exports = {
     'unicode-bom': ['warn', 'never'],
     'use-isnan': 'warn',
     'valid-typeof': 'warn',
+    'no-restricted-properties': ['warn', {
+      object: 'require',
+      property: 'ensure',
+      message: 'Please use import() instead.'
```

Comment:
One pitfall is we won't want to merge that doc until 0.10 or we'll get tens of issues claiming the feature doesn't work. 😄